hal/ia32/exceptions.c

```diff
@@ -164,14 +167,15 @@ void hal_exceptionsDumpContext(char *buff, exc_context_t *ctx, int n)
 	i += hal_i2s("\ndr6=", &buff[i], ctx->dr6, 16, 1);
 	i += hal_i2s(" dr7=", &buff[i], ctx->dr7, 16, 1);
 
-	__asm__ volatile
-	(" \
-		xorl %%eax, %%eax; \
-		movl %%cr2, %%eax; \
-		movl %%eax, %0"
-	:"=r" (ss)
+	/* clang-format off */
+	__asm__ volatile (
+		"xorl %%eax, %%eax\n\t"
+		"movl %%cr2, %%eax\n\t"
+		"movl %%eax, %0"
+	: "=r" (ss)
```

Comment:
```suggestion
	/* clang-format off */
	__asm__ volatile (
		"movl %%cr2, %0\n\t"
	: "=r" (ss)
```
and remove `eax` from the clobber list